config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
   # Defines the root path route ("/")
   # root "articles#index"
+  root 'users#index'
+  get 'users/:id', to: 'users#show'
+  get 'users/:id/posts', to: 'posts#index'
+  get 'users/:id/posts/:post_id', to: 'posts#show'
```

Comment:
- [ ] [OPTIONAL] Kindly write the routes in this way:
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

as it's cleaner and easier to read and maintain.

Reply:
Thanks @gemmen29